src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -175,12 +175,15 @@ public boolean check() throws Exception {
 				continue;
 			}
 
-			// If the user isn't in a white-listed world, commands will not run here.
-			if ( !getConfig().getStringList( "whitelist-worlds" ).contains( player.getWorld().getName() ) ) {
+			//Check for enabled world whitelisting in config
+			if(getConfig().getBoolean("enable-world-whitelist")){

```

Comment:
At this point we'd be adding a config option for a config option. Probably not the best idea.